packages/teams-js/src/private/teams.ts

```diff
@@ -82,4 +85,77 @@ export namespace teams {
   export function isSupported(): boolean {
     return runtime.supports.teams ? true : false;
   }
+
+  export namespace fullTrust {
```

Comment:
Can you please add this above the namespace declaration?

/**
 * @hidden
 * ------
 * Hide from docs
 *
 * @internal
 */

Reply:
Ugh formatting. But you get the gist.

Reply:
I'm sorry if my formatting led you astray. Can you please make sure it matches the same format as the comments as on lines 103-105 (don't need line 106)?

E.g., we don't need the extra lines here on 96, 98 and the H in "Hide" should be capitalized.